include/deal.II/grid/grid_tools.h

```diff
@@ -2632,7 +2632,36 @@ namespace GridTools
   copy_material_to_manifold_id(Triangulation<dim, spacedim> &tria,
                                const bool compute_face_ids = false);
 
-
+  /**
+   * Propagate manifold indicators associated to the cells of the Triangulation
+   * @p tria to their co-dimension one and two objects.
+   *
+   * This function sets the @p manifold_id of shared faces and edges to the value
+   * returnd by the @p disambiguation_function method, called with the set of
+   * manifold indicators of the cells that share the same face or edge.
```

Comment:
The documentation doesn't say it explicitly, but I suspect that it does this for *all* faces and edges, interior or at the boundary, whether or not they already have manifold ids, right? This may be worthwhile saying explicitly.